src/amo/components/AddonMeta/styles.scss

```diff
@@ -60,6 +59,11 @@
     &:visited {
       color: $black;
     }
+
+    &:active,
```

Comment:
I'm not sure why I have to repeat these here, and they're not simply inherited from the definition for `a:active, a:hover` below, but without this explicit override the link remains black when being hovered over.

Reply:
I don't thin the below styles do anything..?

it looks like the style is basically this:

```css
a.AddonMeta-reviews-content-link a.active {
 // styles
}
```

it's styling for nested links so I that shouldn't work b/c that shouldn't be valid html.. if that make sense. I think you can probably remove the below styles

Reply:
oops/sorry..disregard my above comment. I misread this and thought this was nested...